CHANGE_LOG.md

```diff
@@ -13,6 +13,7 @@ All notable changes to this project will be documented in this file.
 - Fixed `ariaTelemetryLogger` not updating `collectorUri`
 - Update `environmentVersion` to be `prod` by default in `defaultInternalTelemetryData`
 - Do not log OOOH errors as WidgetLoadFailure
+- Fix issue with makrdown render which exposed the chat for XSS attacks.
```

Comment:
I believe all these should go into chat-components section

Reply:
XSS change was made in chat-widget